### DISCUSSION TITLE
Steam Deck as ToyPad

### DISCUSSION THREAD

**Discussion by Aspenini** [Berny23/LD-ToyPad-Emulator#220] [Ideas]
_Idk if it's possible or not_, but since the Steam Deck runs **Linux** couldn't you use it to run the program and act as a graphical interface to select characters?

**Comment by DomanDragon** [comment]
It's not about whether the device supports Linux, but whether or not it has a dual role USB port. Fortunately, with some [tweaking in the BIOS](https://github.com/Frederic98/GadgetDeck), it should work as a USB gadget.

However, the Wii U doesn't have enough power to keep the Steam Deck charged. You'll probably want to just use the Deck to use the browser UI.

**Reply by DomanDragon** [reply]
Okay, update:

I tried it myself, and remembered that SteamOS doesn't have apt, it uses pacman and has an immutable file system. You'll need to use a different distro if you want to set it up.
